src/argilla/server/alembic/versions/7cbcccf8b57a_create_metadata_properties_table.py

```diff
@@ -30,31 +30,21 @@
 
 
 def upgrade() -> None:
-    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
@gabrielmbmb I will update the PR so the migration is automatically made by Alembic.

Reply:
Done.

I have changed the migration so we use alembic automatic migration and added some missing stuff on the models. 

You can see the changes on this commit: https://github.com/argilla-io/argilla/pull/3952/commits/639f07ceef42bef5dca1ee23771bc148ca17cee9